envoy/Dockerfile

```diff
@@ -22,12 +22,14 @@ ENV GO_VERSION 1.10.3
 # Build dependencies
 #
 RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get upgrade -y --no-install-recommends \
```

Comment:
What does this line do? It seems like the line immediately below but without any packages specified.

Reply:
one is `upgrade` the other is `install` :smile: